app/models/manageiq/providers/vmware/infra_manager/provision/cloning.rb

```diff
@@ -54,18 +57,17 @@ def dest_resource_pool
     resource_pool = ResourcePool.find_by(:id => respool_id) unless respool_id.nil?
     return resource_pool unless resource_pool.nil?
 
-    cluster = dest_host.owning_cluster
-    cluster ? cluster.default_resource_pool : dest_host.default_resource_pool
+    dest_cluster ? dest_cluster.default_resource_pool : dest_host.default_resource_pool
   end
 
   def dest_folder
     folder_id = get_option(:placement_folder_name)
     return EmsFolder.find_by(:id => folder_id) if folder_id
 
-    host_dc = dest_host.parent_datacenter || dest_host.ems_cluster.parent_datacenter
+    dc = dest_cluster ? dest_cluster.parent_datacenter : dest_host.parent_datacenter
```

Comment:
A host always has at least a `ComputeResource` but for some reason we only inventory `ClusterComputeResource`, this could be simplified to just `dc = dest_cluster.parent_datacenter` if we change that, might revisit later.